### PR TITLE
NXP-28862: modify date format to show time for nuxeo document history and audit

### PR DESCRIPTION
**Pull request by andreacornaglia**

### CODE REVIEW

File: elements/nuxeo-document-history/nuxeo-document-history.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ Polymer({
           <template>[[_formatActivity(item.eventId)]]</template>
         </nuxeo-data-table-column>
         <nuxeo-data-table-column name="[[i18n('documentHistory.date')]]" sort-by="eventDate">
-          <template><nuxeo-date datetime="[[item.eventDate]]"></nuxeo-date></template>
+          <template><nuxeo-date datetime="[[item.eventDate]]" format="LLL"></nuxeo-date></template>
```
Comment (nmpcunha) [inline elements/nuxeo-document-history/nuxeo-document-history.js:74]: With this change the tooltip will show exactly the same as the span.
Is this a pattern we want to promote from an UX perspective? (cc @carolinagouveia , @semisse )

Comment (andreacornaglia) [reply elements/nuxeo-document-history/nuxeo-document-history.js:74]: by span you mean tooltip? I think so.

Comment (nmpcunha) [reply elements/nuxeo-document-history/nuxeo-document-history.js:74]: I was just stating that we will have the same content information on both: span and tooltip.

Comment (nmpcunha) [reply elements/nuxeo-document-history/nuxeo-document-history.js:74]: Screenshot 2020-05-04 at 10 49 42

Comment (andreacornaglia) [reply elements/nuxeo-document-history/nuxeo-document-history.js:74]: hiding tooltip when values are same for date: https://github.com/nuxeo/nuxeo-elements/pull/233/files

**Pull request merged**